helm/botkube/e2e-test-values.yaml

```
@@ -106,6 +106,13 @@ settings:
 
 extraAnnotations:
   botkube.io/disable: "true"
+
+## Parameters for anonymous analytics collection.
+analytics:
+  # -- If true, sending anonymous analytics is disabled. To learn what date we collect,
+  # see [Privacy Policy](https://botkube.io/privacy#privacy-policy).
+  disable: true
```

We can remove the comments:

```suggestion
analytics:
  disable: true
```

---

I left them on purpose.

---

As it's an important setting.

---

I just think of consistency, as we don't have other comments anywhere else in this file, as they are overrides. Especially that the comments are copied with special characters (e.g. `# --`) and Markdown syntax which are used to generate our docs 🙂 We have all values documented in the main `values.yaml` file. 

Of course this is non-blocking comment, but it would be great to sort it out. I see two main options:
- remove these comments
- remove the syntax for docs:

```
# Parameters for anonymous analytics collection.
analytics:
  # If true, sending anonymous analytics is disabled. To learn what date we collect,
  # see Privacy Policy: https://botkube.io/privacy#privacy-policy.
```

Please apply the changes before merge 🙂